src/software/thunderscope/binary_context_managers.py

```diff
@@ -116,6 +124,18 @@ def __enter__(self):
 
         return self
 
+    def __restart__(self):
+        "Restarts full system."
+
+        while not is_cmd_running(
```

Comment:
this will only restart full system once, i think we want a loop that always runs with an if statement that checks

so 

psudeo code

while True:
   if is_cmd_running():
       run_cmd here
sleep(1)